sdk/core/System.ClientModel/src/Pipeline/ClientPipeline.cs

```diff
@@ -104,10 +112,7 @@ public static ClientPipeline Create(ClientPipelineOptions options, params Pipeli
 
         int perTryIndex = index;
 
-        // TODO: Buffering policy will come in a later PR.
-        //TimeSpan networkTimeout = options.NetworkTimeout ?? PipelineResponse.DefaultNetworkTimeout;
-        //ResponseBufferingPolicy bufferingPolicy = new(networkTimeout);
-        //policies[index++] = bufferingPolicy;
+        policies[index++] = new ResponseBufferingPolicy();
```

Comment:
Since this ctor no longer takes any arguments, should we create a static instance and use that, like we do with some other policies.

Reply:
Let me look -- I think that should work.